src/structured_config.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
```

Comment:
**thought:** This file is named `structured_config`, which is a good name.

Do we imagine having a `structured_actions` and a `structured_relationdata` in the future? 
Or would those be added to the same file?

Just wondering if there's a more specific name for the file